roles/seqreports/tasks/pre_reqs.yml

```diff
@@ -1,5 +1,13 @@
 ---
 
+- name: fetch value for virtual env environment variable
```

Comment:
Could you explain how the virtual env is set up and what it is used for?

Reply:
In the following task, I call a python script, that only works if the bootstrap virtual environment is activated. Apparently, when calling this script from an ansible task, even if I currently has activated my virtualenv "miarkaenv" when triggering the playbook, it is not used unless I explicitly call "source miarkaenv" from the task execution line. I wanted to do this a little bit more generic, and not bind it the the actual name "miarkaenv". The task above activates whatever virtualenv the caller has activated when running this playbook.